ClassySharkWS/src/com/google/classyshark/updater/models/Release.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.classyshark.updater.models;
+
+
+import com.google.classyshark.Version;
+import com.google.gson.annotations.SerializedName;
+import com.sun.istack.internal.Nullable;
+
```

Comment:
Please add here a comment, about what does this class mean and how we do versioning.